src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsList.tsx

```diff
@@ -41,63 +65,109 @@ const ConditionsList: React.FC<IConditionsListProps> = (props) => {
+    const handleDismissFlyout = useCallback(() => {
+        setCalloutInfo({
+            calloutType: CalloutInfoType.inactive,
+            isOpen: false,
+            selectedCondition: null,
+            selectedTarget: ''
+        });
+    }, []);
+
+    const getOverflowMenuItems = useCallback(
+        (conditionId: string): IContextualMenuItem[] => [
+            {
+                key: `${conditionId}-edit-menu-item`,
+                text: t('3dSceneBuilder.visualRuleForm.editCondition'),
+                iconProps: {
+                    iconName: 'Edit'
+                },
+                onClick: () => {
+                    setCalloutInfo({
+                        calloutType: CalloutInfoType.edit,
+                        isOpen: true,
+                        selectedCondition: valueRanges.find(
+                            (vr) => vr.id === conditionId
+                        ),
+                        selectedTarget: `#${LIST_KEY}`
+                    });
+                },
+                data: {
+                    id: conditionId
+                }
             },
-            data: {
-                id: conditionId
+            {
+                key: `${conditionId}-delete-menu-item`,
+                text: t('3dSceneBuilder.visualRuleForm.deleteCondition'),
+                iconProps: {
+                    iconName: 'Delete'
+                },
+                onClick: (_ev, item) => {
+                    onDeleteCondition(item.data.id);
+                    handleDismissFlyout();
+                },
+                data: {
+                    id: conditionId
+                }
             }
+        ],
+        [handleDismissFlyout, onDeleteCondition, t, valueRanges]
+    );
+
+    const getConditionItems = useCallback(
+        (
+            valueRanges: IValueRange[],
+            expressionType: IExpressionRangeType
+        ): ICardboardListItem<Condition>[] => {
+            const conditions = transformValueRangesIntoConditions(
+                valueRanges,
+                expressionType
+            );
+            const viewModel: ICardboardListItem<Condition>[] = conditions.map(
+                (condition) => {
+                    return {
+                        item: condition,
+                        ariaLabel: `Condition for ${condition.primaryText}`,
+                        textPrimary: condition.primaryText,
+                        textSecondary: condition.secondaryText,
+                        overflowMenuItems: getOverflowMenuItems(condition.id),
+                        onClick: () => {
+                            setCalloutInfo({
+                                calloutType: CalloutInfoType.edit,
+                                isOpen: true,
+                                selectedCondition: valueRanges.find(
+                                    (vr) => vr.id === condition.id
+                                ),
+                                selectedTarget: `#${LIST_KEY}`
```

Comment:
the flyout should open next to the item you are editing. Right now it only seems to open in a static location.

Reply:
This requires an edit to cardboard list, currently the list items don't take in an id as props and I can't target them. I can add the prop and change the target in a following PR.